001-core/constants.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Automattic\VIP\Core\Constants;
+
+/**
+ * We use HyperDB, so DB_* constants are not automatically defined.
+ * We define them here in case any customer code is referencing them to avoid fatals errors in PHP 8+.
+ * As a best practice, these constants should not be used directly (`_doing_it_wrong()`).
+ *
+ * @param Object $wpdb
+ * @return void
+ */
+function vip_define_db_constants( $wpdb ) {
```

Comment:
The function doesn't need the `vip_` prefix, since it is namespace sufficiently to avoid a clash.

Also, can we at least add the return type `: void`?

Reply:
Done.